binding/query_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Illia Dimura. All rights reserved.
```

Comment:
// Copyright 2022 Gin Core Team. All rights reserved.
// Use of this source code is governed by a MIT style
// license that can be found in the LICENSE file.